README.md

```diff
@@ -1,25 +1,49 @@
 # Spdx-Java-Library
 
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.spdx/java-spdx-library/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.spdx/java-spdx-library)
+[![javadoc](https://javadoc.io/badge2/org.spdx/java-spdx-library/javadoc.svg)](https://javadoc.io/doc/org.spdx/java-spdx-library)
 ![Java CI with Maven](https://github.com/spdx/Spdx-Java-Library/workflows/Java%20CI%20with%20Maven/badge.svg)
 
-Java library which implements the Java object model for SPDX and provides useful helper functions.
-
-The API documentation is available at:
-<https://spdx.github.io/Spdx-Java-Library/>
-
-## Code quality badges
-
 [![Bugs](https://sonarcloud.io/api/project_badges/measure?project=java-spdx-library&metric=bugs)](https://sonarcloud.io/dashboard?id=java-spdx-library)
 [![Security Rating](https://sonarcloud.io/api/project_badges/measure?project=java-spdx-library&metric=security_rating)](https://sonarcloud.io/dashboard?id=java-spdx-library)
 [![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=java-spdx-library&metric=sqale_rating)](https://sonarcloud.io/dashboard?id=java-spdx-library)
 [![Technical Debt](https://sonarcloud.io/api/project_badges/measure?project=java-spdx-library&metric=sqale_index)](https://sonarcloud.io/dashboard?id=java-spdx-library)
 
+Spdx-Java-Library is a Java library which implements the Java object model for SPDX and provides useful helper functions.
+
+The library is available in Maven Central as
+[`org.spdx:java-spdx-library`](https://search.maven.org/artifact/org.spdx/java-spdx-library)
+(note the order of the word "java-spdx").
+
+If you are using Maven, you can add the following dependency in your POM file:
+
+```xml
+<dependency>
+  <groupId>org.spdx</groupId>
+  <artifactId>java-spdx-library</artifactId>
+  <version>(,2.0]</version>
+</dependency>
+```
+
+See the [GETTING-STARTED.md](GETTING-STARTED.md) file for how to get started in different scenarios.
+
+## Table of Contents
+
+- [Library Version Compatibility](#library-version-compatibility)
+- [Storage Interface](#storage-interface)
+  - [Storage Interface Usage](#storage-interface-usage)
+- [Multi-Threaded Considerations](#multi-threaded-considerations)
+- [Configuration Options](#configuration-options)
+- [Initialization](#initialization)
+- [Update for New Versions of the Spec](#update-for-new-versions-of-the-spec)
+- [API Documentation](#api-documentation)
+- [Contributing](#contributing)
+
 ## Library Version Compatibility
 
 Library version 2.0.0 and higher is not compatible with previous versions of the library due to breaking changes introduced in SPDX 3.0.
 
-The library does support the spec versions 2.X and 3.X.
+The library does support the specification versions 2.X and 3.X.
```

Comment:
I'd suggest:
> The library supports the ...

Reply:
I guess the "does" is here to put the stress. To contrast the breaking changes info in the 1st para. I have revise this a bit, putting "however" to link two paras and signal the contrastness.